docs-v2/integrations/all/greenhouse-basic/connect.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Greenhouse - How do I link my account?
+sidebarTitle: Greenhouse
+---
+
+
+# Overview
+
+To authenticate with Greenhouse, you need one key piece of information:
+1. **API key** - Your Greenhouse Access Key
+
+This guide will walk you through finding or creating those credentials within Greenhouse.
+
+### Prerequisites:
+
+- You must set up an account with Greenhouse.
+
+#### Step 1: Finding Greenhouse API Key
+
+1. Log into your Greenhouse account and click the `Configure` ⚙️ icon on your navigation bar, then click Dev Center.
+2. Click API Credential Management.
+
+<img src="/integrations/all/greenhouse-basic/greenhouse_1.png" />
+
+<img src="/integrations/all/greenhouse-basic/greenhouse_2.png" />
+
+3. Click Create New API Key.
+
+4. Make the following selections:
+    - API type: Job board
+    - Description: Give your API key a descriptive name
+
+5. When finished, click `Create` to generate the API key.
+
+6. After the key is created, click `Copy`. Paste the API key in a secure location.
+
+7. Click `I have stored the API Key`.
+
+8. Note your greenhouse.io domain in your browser.
```

Comment:
The steps are not linked in `providers.yaml` and it's missing the step 1 about the domain